.github/PULL_REQUEST_TEMPLATE/register_did_method.md

```diff
@@ -0,0 +1,47 @@
+## Register a DID Method
+
+### Required Registration Criteria
+
+Please confirm:
+
+- [ ] The DID Method specification defines the identifer syntax.
+- [ ] The DID Method specification defines the supported DID Operations.
+- [ ] The DID Method specification contains a Privacy and Security Considerations.
+
+### Optional Registration Criteria
+
+Consider ensuring:
+
+- [ ] The registration entry contains contact information.
+- [ ] The registration entry contains registry information.
+
+### Registration Process
+
+In order to register a New DID Method, you must add a json file to [./methods](./methods).
+
+Here is an example registration entry:
+
+```jsonc
+{
+  // These fields are required
+  "name": "example",
+  "status": "registered",
+  "specification": "https://w3c-ccg.github.io/did-spec/",
+  // These fields are optional
+  "contactName": "W3C Credentials Community Group",
+  "contactEmail": "",
+  "contactWebsite": "",
```

Comment:
This is for future PRs, just writing it down while I'm thinking about it:

We should reject registrations that don't provide a `contactEmail` address. It's a huge pain trying to figure out how to get in touch w/ these DID Method authors.

Reply:
we can very easily do that, now that we have: 

https://github.com/w3c/did-spec-registries/pull/361/files#diff-ea6b2d66de4a221273936ab35afb0fae0f8ace048d29642c2440dcfe59c206e1R6

You can request changes on that PR, to add these fields to the required section... however, they will still be empty for many existing entries....  I think its best we not co-mingle that issue with updating the process, we can address it after we have changed the process, adding it to the current list of changes will make these PRs more contentious and harder to review.

Reply:
I opened: https://github.com/w3c/did-spec-registries/issues/362 Please keep the discussion to that issue to keep the PR free from it.

Reply:
I'm resolving this thread so folks know to contribute to the issue regarding this conversation.